File: kyu-5/_0003__RGB_To_Hex_Conversion/solution.js
const rgb = (r, g, b) => [r, g, b].map((n) => (n >= 255 && 'FF') || (n <= 0 && '00') || n.toString(16).toUpperCase().padStart(2, 0)).join('');

console.log(rgb(255, 255, 255)); // "FFFFFF"
console.log(rgb(0, 0, 0)); // "000000"
console.log(rgb(3, 3, 3)); // "030303"

File: kyu-5/_0003__RGB_To_Hex_Conversion/solution.rb
def rgb(r, g, b)
  [r, g, b].map! do |c|
    next "FF" if c >= 255
    next "00" if c <= 0
    c.to_s(16).upcase.rjust(2, '0')
  end.join()
end


p rgb(255, 255, 255)
p rgb(0, 0, 0)
p rgb(3, 3, 3)